Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -123,77 +123,115 @@ The list should be maintained in alphabetic order. DELETE This Row
 
-* **Who is the target audience?** 
+#### Target Audience
 
 The target audience for this are technical stakeholders (e.g. software architects, developers, and maintainers) who ideally can use this as a methodology so that they can understand the characteristics of their software solution and minimize the associated emissions.
 
-This standard uses a consequential approach to calculating the real world emissions associated with software. This approach measures the total change in global emissions associated with a particular piece of software.
+
+## Methodology Summary 
+
+This standard can be used to calculate the real-world emissions associated with software by measuring the total change in global emissions associated with a particular piece of software.
 
 Electricity has a carbon intensity depending on where and when it is consumed. An intensity is a rate. It has a numerator and a denominator. A rate provides you with helpful information when considering the growth of a software product and allows for the computation of a marginal rate.
 
-A carbon intensity is a rate of carbon emissions compared against another measure. 
+To calculate the carbon intensity we need the following:
 
-To calculate the carbon intensity we need to decide two things.
+`O`= Operational emissions of a given piece of software
 
-`C / R`
+`E` = Embodied emissions of a given piece of software
 
-`C` = The total amount of carbon the software is emitting over a time period.
+**These are used to calculate total carbon emissions (`C`):**
 
-`R` = The other measure we will be comparing against.
+`C` = Total amount of carbon the software is emitting over a time period.
 
-## Taxonomy of Green Software
+`C = O + Embodied `
+
+
+## Taxonomy & Terminolopgy of Green Software
 
 Green Software broadly addresses emissions in these categories.
 
-***Operational Emissions:***
+***Operational Emissions (`O`):***
+To calculate the operational emissions associate with software, multiply the electricity consumption of the hardware the software is running on by the regional, granular marginal emissions rate. Because this standard uses a consequential approach, marginal emissions rates should be used for electricity consumption. The marginal emissions rate reflects the change in emissions assoicated with a change in demand. 
+
+Carbon aware software will optimize the timing and location of operation to minimize emissions associated with operation. This could consist of moving computation to regions with cleaner grid emissions or delaying jobs to cleaner periods (or a combination of both). Energy efficient software will also run on hardware that requires less energy to operate or the software can be re-architected to require less energy to execute. All combined, these effects should be reflected in the total operational emissions. 
+
+### Energy Measurement (`E`) 
```

Comment:
I just realised we are also using `E` in the section above for Embodied. Thoughts on using `M` for Embodied and `E` for Energy?

Reply:
I had used a lower case 'e' for energy as distinct from 'E' for embodied emissions but that might be confusing.

Reply:
M seems to be a better choice to prevent confusion 😊